.pre-commit-config.yaml

```diff
@@ -30,7 +30,7 @@ repos:
     rev: 3.9.2
     hooks:
     -   id: flake8
-        args: [--max-line-length, "120", "--extend-exclude=tests/**"]
+        args: [--max-line-length, "120", "--extend-exclude=tests/**", "--ignore=F601,W503 *"]
```

Comment:
Address these issues rather than globally ignoring the errors.

The first one suggests that you've created an invalid dictionary, with duplicate keys. See [this example for the second code](https://www.flake8rules.com/rules/W503.html)